config/routes.rb

```diff
@@ -0,0 +1,21 @@
+Rails.application.routes.draw do
+  apipie
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  # Defines the root path route ("/")
+  root "apipie/apipies#index"
+  get 'users/login-register', to: 'users#login_register'
+ 
+  resources :users do
+    get 'meetings', to: 'user_details#meetings'
+    resources :user_details
+    resources :reservations
```

Comment:
- [ ] Since this creates all available routes, I would advise you to specify which ones you are really using, so that they aren't created needlessly.